app/src/pages/inside/settingsPage/settingsTabs/autoAnalysisTab/autoAnalysisTab.scss

```diff
@@ -0,0 +1,109 @@
+.settings-tab-content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  padding: 15px;
+  @media (min-width: $SCREEN_MD_MAX_DASHBOARD) {
+    width: 1040px;
+    max-width: 1040px;
+    margin: 0 auto;
+  }
+}
+
+.form-group-container {
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+  width: 100%;
+  margin-bottom: 30px;
+  @media (max-width: $SCREEN_XS_MAX) {
+    flex-direction: column;
+    align-items: flex-start;
+    margin-bottom: 15px;
+  }
+}
+
+div.switcher-wrapper {
```

Comment:
Do we really need complicated selector? Why not to use just selector by className?